message.go

```diff
@@ -36,6 +36,23 @@ const (
 )
 
 type Props map[string]string
+
+func (props *Props) GetIntValue(key string) (int64, error) {
+	val, ok := (*props)[key]
+	if !ok {
+		return 0, fmt.Errorf("property %s not found", key)
+	}
+	return strconv.ParseInt(val, 10, 64)
```

Comment:
I am thinking... properties are mostly optional and this gives us a hard time differentiating between valid property and simple not set property. For instance, if someone sent 3s instead of 3 in duration property for out_lvl_switch level set command maybe we should emit an error instead of ignoring it silently and just changing level without any duration.

There are I think generally four ways of doing it. 
1) Return type of value, bool and error (complex solution)
2) Return type of value and error (your current solution)
3) Return type of value and bool (we don't care about errors, just if there is a valid property set)
4) Return type only (we don't care about errors, we return zero-value if there is no valid property set)
5) Mix 4 with 1. Return errors when there is casting issue, return zero-value if property is not set

Which do you think makes most sense here?